docs/src/components/BuiltinSearch/styles.module.css

```diff
@@ -0,0 +1,100 @@
+.versionTag a:link {
+  color: white;
+}
+
+.versionTag {
+  background-color: slategrey;
+  color: white;
+  font-weight: bold;
+  font-size: 0.8rem;
+  padding: 0.1rem 0.2rem;
+  border-radius: 0.2rem;
+}
+
+.versionTag a:visited {
+  color: white;
+}
+
+.wasmTag {
+  background-color: seagreen;
+  color: white;
+  font-weight: bold;
+  font-size: 0.8rem;
+  padding: 0.1rem 0.2rem;
+  border-radius: 0.2rem;
+}
+
+.sdkTag {
+  background-color: darkgoldenrod;
+  color: white;
+  font-weight: bold;
+  font-size: 0.8rem;
+  padding: 0.1rem 0.2rem;
+  border-radius: 0.2rem;
+  white-space: nowrap;
+}
+
+.categoryTag {
+  color: white;
+  font-weight: bold;
+  font-size: 0.8rem;
+  padding: 0.1rem 0.2rem;
+  border-radius: 0.2rem;
+  white-space: nowrap;
+}
+
+.categoryTag a:link {
+  color: white;
+}
+.categoryTag a:visited {
+  color: white;
+}
+.searchBar {
+  display: flex;
+  background-color: inherit;
+  color: white;
+  font-weight: bold;
+  font-size: 0.8rem;
+  padding: 0.1rem 0.2rem;
+  border-radius: 0.2rem;
+  white-space: nowrap;
+}
+.searchBar input {
+  display: flex;
+  background-color: inherit;
+  color: white;
+  font-weight: bold;
+  font-size: 0.8rem;
+  padding: 0.1rem 0.2rem;
+  border: none;
+  border-bottom: 0.1rem solid grey;
+}
+.searchBar select {
+  display: flex;
+  background-color: inherit;
+  color: white;
```

Comment:
I think we can use the docusaurus theme variables here to make sure this is visible on light mode too.